ee/src/main/java/org/jboss/as/ee/metadata/EJBClientDescriptorMetaData.java

```diff
@@ -346,4 +353,16 @@ public int hashCode() {
             return outboundConnectionRef != null ? outboundConnectionRef.hashCode() : 0;
         }
     }
+
+    public class HttpConnectionConfiguration {
```

Comment:
This should be final and presumably static.

Reply:
fixed